tmh_registry/users/models.py

```diff
@@ -17,12 +26,12 @@ class MedicalPersonnel(models.Model):
 
     level = models.CharField(
         max_length=255,
-        choices=[("LEAD_SURGEON", "Lead Surgeon")],
-        default="Lead Surgeon",
+        choices=LEVEL_CHOICES,
+        default=LEAD_SURGEON,
     )
 
     class Meta:
         verbose_name_plural = "Medical Personnel"
 
     def __str__(self):
-        return self.user.username
+        return f"({self.level}) {self.user.username}"
```

Comment:
@dkoutsonikolis What about `self.get_level_display()` instead?

Reply:
I changed it - not sure if there is a better way.

Reply:
Never mind, alll good. :)